pkg/chunked/internal/compression.go

```diff
@@ -99,7 +100,7 @@ const (
 	// FooterSizeSupported is the footer size supported by this implementation.
 	// Newer versions of the image format might increase this value, so reject
 	// any version that is not supported.
-	FooterSizeSupported = 56
+	FooterSizeSupported = 64
```

Comment:
we won't even hit this code path as all the information we need is really stored in the OCI annotations.

I am not really sure if we want to support this mechanism.  I've added it because estargz uses it, and to have a way to retrieve the TOC if we have only the layer file